js/serverConnectionAPI.js

```diff
@@ -0,0 +1,29 @@
+function getOffersFromServer(onSuccess, onFail) {
+  fetch('https://26.javascript.pages.academy/keksobooking/data')
+    .then((response) => response.json())
+    .then((serverOffers) => {
+      onSuccess(serverOffers);
+    })
```

Comment:
```suggestion
    .then(onSuccess)
```

зачем создавать. новую функцию, если можно передать ссылку на уже готовую?